src/_playground/Routes.js

```diff
@@ -327,10 +342,24 @@ export class Routes extends Component {
             );
         });
 
+        const sideBarClasses = classnames(
+            'frDocs-Sidebar',
+            {
+                'sidebar-visibility': !this.state.showSideNav
```

Comment:
Should we follow the same naming convention the docs have in general? Something like `frDocs-Sidebar__visibility`?